src/transformers/models/plbart_multi/tokenization_plbart_multi.py

```diff
@@ -0,0 +1,304 @@
+        "uclanlp/plbart-single_task-strong-generation": "https://huggingface.co/uclanlp/plbart-single_task-strong-generation/resolve/main/sentencepiece.bpe.model",
+        "uclanlp/plbart-single_task-strong-summarization": "https://huggingface.co/uclanlp/plbart-single_task-strong-summarization/resolve/main/sentencepiece.bpe.model",
+        "uclanlp/plbart-single_task-weak-generation": "https://huggingface.co/uclanlp/plbart-single_task-weak-generation/resolve/main/sentencepiece.bpe.model",
+        "uclanlp/plbart-single_task-weak-summarization": "https://huggingface.co/uclanlp/plbart-single_task-weak-summarization/resolve/main/sentencepiece.bpe.model",
+    }
+}
+
+PRETRAINED_POSITIONAL_EMBEDDINGS_SIZES = {
+    "uclanlp/plbart-multi_task-all": 1024,
+    "uclanlp/plbart-multi_task-compiled": 1024,
+    "uclanlp/plbart-multi_task-dynamic": 1024,
+    "uclanlp/plbart-multi_task-go": 1024,
+    "uclanlp/plbart-multi_task-interpreted": 1024,
+    "uclanlp/plbart-multi_task-java": 1024,
+    "uclanlp/plbart-multi_task-js": 1024,
+    "uclanlp/plbart-multi_task-php": 1024,
+    "uclanlp/plbart-multi_task-python": 1024,
+    "uclanlp/plbart-multi_task-ruby": 1024,
+    "uclanlp/plbart-multi_task-static": 1024,
+    "uclanlp/plbart-multi_task-strong": 1024,
+    "uclanlp/plbart-multi_task-weak": 1024,
+    "uclanlp/plbart-single_task-all-generation": 1024,
+    "uclanlp/plbart-single_task-all-summarization": 1024,
+    "uclanlp/plbart-single_task-compiled-generation": 1024,
+    "uclanlp/plbart-single_task-compiled-summarization": 1024,
+    "uclanlp/plbart-single_task-dynamic-generation": 1024,
+    "uclanlp/plbart-single_task-dynamic-summarization": 1024,
+    "uclanlp/plbart-single_task-en_go": 1024,
+    "uclanlp/plbart-single_task-en_java": 1024,
+    "uclanlp/plbart-single_task-en_js": 1024,
+    "uclanlp/plbart-single_task-en_php": 1024,
+    "uclanlp/plbart-single_task-en_python": 1024,
+    "uclanlp/plbart-single_task-en_ruby": 1024,
+    "uclanlp/plbart-single_task-go_en": 1024,
+    "uclanlp/plbart-single_task-interpreted-generation": 1024,
+    "uclanlp/plbart-single_task-interpreted-summarization": 1024,
+    "uclanlp/plbart-single_task-java_en": 1024,
+    "uclanlp/plbart-single_task-js_en": 1024,
+    "uclanlp/plbart-single_task-php_en": 1024,
+    "uclanlp/plbart-single_task-python_en": 1024,
+    "uclanlp/plbart-single_task-ruby_en": 1024,
+    "uclanlp/plbart-single_task-static-generation": 1024,
+    "uclanlp/plbart-single_task-static-summarization": 1024,
+    "uclanlp/plbart-single_task-strong-generation": 1024,
+    "uclanlp/plbart-single_task-strong-summarization": 1024,
+    "uclanlp/plbart-single_task-weak-generation": 1024,
+    "uclanlp/plbart-single_task-weak-summarization": 1024,
+}
+
+FAIRSEQ_LANGUAGE_CODES = [
+    "java",
+    "python",
+    "en_XX",
+    "javascript",
+    "php",
+    "ruby",
+    "go",
+]
+
+
+class PLBartMultiTokenizer(XLMRobertaTokenizer):
```

Comment:
Is this one really that different that we can't put it into the same class?

Reply:
The two tokenizers have different language codes.